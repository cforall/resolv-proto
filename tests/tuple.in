1 f-c 2
1 1 f-cc 3
1 1 1 f-ccc 1

g 1 1 1 1
%%
g( f(2) f(2) )

