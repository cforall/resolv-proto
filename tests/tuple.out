[Void / (0,0,0,0,2)] g( f-cc( 2 => 3 ) f-cc( 2 => 3 ) )
